index.html

```diff
@@ -0,0 +1,35 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Portfolio</title>
+    <link rel="stylesheet" href="CSS/styles.css">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+    <link href="https://fonts.googleapis.com/css2?family=Playfair+Display&family=Poppins&display=swap" rel="stylesheet">
+
+</head>
+<body>
+    <div class="wrapper">
+        <header class="app-header">
+            <img class="graphic1" src="img/image_geometry_1.svg" alt="Graphic art">
+            <img class="menu-icon" src="img/ic_menu.svg" alt="Menu Icon">
+        </header>
+    
+        <div class="content">
+            <div class="name">
+                <p class="first name">Oshoke<br> Okolo</p>
+            </div>
+            <p class="introduction">Hello! I am a software developer! I can help you build a product, feature or website. Taka a look of my works.
+                If you like what you see and have a project you need coded, don’t hesitate and contact me.
+            </p>
+            <button type="button" class="collaborate-btn start">
+                Start Collaboration
+            </button>
+            <img class="graphic2" src="img/image_geometry_2.svg" alt="Graphic art">
+        </div>
```

Comment:
- [ ] Kindly make this a `section` tag to wrap the headline section instead of a div tag. It's for the purpose of making your code semantic which will help screen readers and make your code meaningful. It's required you make this a `section` tag. Please fix that 👍